addons/xpaas-templates/xpaas-templates.addon

```diff
@@ -0,0 +1,102 @@
+# Name: xpaas
+# Description: Imports xPaaS templates
+# Url: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_examples/files/examples/v3.6
+
+oc create -f xpaas-streams/fis-image-streams.json -n openshift
```

Comment:
@hferentschik yes it will, I didn't notice. l am tend to remove fuse completely if that is also part of `xpaas`.